### PR TITLE
Syn 127 add visualization of simulated traces

### PR DESCRIPTION
**Pull request by leny-mi**
## Describe your changes

This PR includes a temorary configuration screen, a simulation confirmation screen and an analysis screen. It is currently filled with placeholders.
- New bottom drawer
- Moved configuration screen
- Added simulation confirmation screen
- Added analysis (statistics) screen for the editor. (Statistics for imported traces are still missing)
- Added preview of the flow (on the bottom drawer, and on the main screen)
- Added Footer to main screen
- Made the add button on main screen a bit prettier 
- QoL changes and refactoring

## Checklist before requesting a review
- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by Volume999** [comment]
image
was that before?

**Comment by Volume999** [comment]
image

I am also not getting any output on the XES file exported but get it to the simulator. I guess it is also not the scope of this project, unless this function works outside of these changes?

{
  "trace": {
    "events": [
      {
        "name": "t1",
        "meta": {}
      },
      {
        "name": "t2",
        "meta": {}
      },
      {
        "name": "t2",
        "meta": {}
      }
    ]
  },
  "exitReason": "noEnabledActivities"
}

**Comment by leny-mi** [comment]
To the last, yeah that's a placeholder. I wanted to implement that as well but we still don't have bulk simulations running. Will do after that is done

**Comment by Volume999** [comment]
The load time is kind of annoying, It's not part of the PR but I think analytics should be optional without such a loading time, what do you think?

**Review by MathiasBoi6 — APPROVED** [review]
Cool interface. tough text can be a bit to small and it is unclear that the analysis is based on random data / not related to model. That said, theses things can fixed later

File: packages/synthchron-tauri/src/components/ProjectCard.tsx [hunk 1]
```diff
@@ -20,34 +32,63 @@ export const ProjectCard: React.FC<ProjectCardProps> = ({
...
         sx={{
-          height: 150,
+          height: 100,
         }}
       >
         <Typography
-          sx={{ fontSize: 14, textTransform: 'capitalize' }}
+          sx={{ textTransform: 'capitalize', mb: 0 }}
           color='text.secondary'
+          fontSize={10}
```
[the remark this answers is not in the document]
Comment (leny-mi) [reply packages/synthchron-tauri/src/components/ProjectCard.tsx:77]: Do you mean the "Petri-Net" and "last edited [...]" lines? Or the petri net preview?

**Review by Volume999 — APPROVED** [review]
lgtm, other things are basically design choices and I don't think are that important

**Pull request merged**